config/exchanges.json

```diff
@@ -1308,7 +1308,7 @@
       43.158267
     ],
     "parsers": {
-      "exchange": "GE.fetch_exchange"
+      "exchange": "RU.fetch_exchange"
```

Comment:
Did you check that the `RU.py` parser returns this exchange data? :)

Reply:
Yes I did!